dev-compose.yaml

```diff
@@ -15,25 +15,44 @@ services:
       - ./init.sql:/docker-entrypoint-initdb.d/init.sql
     ports:
       - "5432:5432"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U letta"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    deploy:
+      resources:
+        limits:
+          memory: 1G
+          cpus: '1.0'
+        reservations:
+          memory: 256M
+          cpus: '0.2'
   letta_server:
-    image: letta/letta:latest
-    hostname: letta
+    image: letta_server
```

Comment:
I did not have permission to pull the image from registry, so I modified the name here.